felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -1296,8 +1334,11 @@ func (m *bpfEndpointManager) markEndpointsDirty(ids set.Set[any], kind string) {
 					}
 				}
 			} else {
-				log.Debugf("Mark host iface dirty, for host %v change", kind)
+				log.Debugf("Mark host iface dirty %v, for host %v change", id, kind)
 				m.dirtyIfaceNames.Add(id)
+				for _, slaveIface := range m.getBondSlaves(id) {
```

Comment:
nit: If you returned the Set you could iterate over the set without creating the slice. In this case it does not matter much since there are going to be a handful of slaves, but in general, I'd advise to watch for this.